### DISCUSSION TITLE
Text section exceeds available space in board

### DISCUSSION THREAD

**Discussion by flashdato** [pschatzmann/arduino-audio-tools#1336] [Q&A] [answered]
I am new on this board I have ESP32 Audio Kit v2.2 and I want to stream audio on (with web). first of all I wanted to start with bluetooth speaker and it is not working 

Sketch uses 1371285 bytes (104%) of program storage space. Maximum is 1310720 bytes.
Global variables use 49428 bytes (15%) of dynamic memory, leaving 278252 bytes for local variables. Maximum is 327680 bytes.
Sketch too big; see https://support.arduino.cc/hc/en-us/articles/360013825179 for tips on reducing it.
text section exceeds available space in board

Compilation error: text section exceeds available space in board

I am using ESP32 Dev Module to upload code on it but it doesnt let me it stops on compile 

should I use another board type?

**Comment by pschatzmann** [comment, marked as answer]
No just select a better suited Partition Scheme: e.g. Hugh App

**Reply by flashdato** [reply]
Thank you so much
